CharlieBackend.Business/Helpers/ResponseMessages.cs

```diff
@@ -1,12 +1,17 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Text;
-
-namespace CharlieBackend.Business.Helpers
+﻿namespace CharlieBackend.Business.Helpers
 {
-    public class ResponseMessages
+    public static class ResponseMessages
     {
         public const string GroupNotFound = "Group not found";
+
         public const string GroupHasNotStudents = "Group hasn't any students";
+
+        public const string IndexNotValid = "Index cannot be less then 1 and greater than 5";
+
+        public static string NotExist(string entityName) => $"{entityName} does not exist ";
+
+        public static string NotValid(string entityName) => $"{entityName} is not valid ";
+
+        public static string NotActive(string entityName) => $"{entityName} is not active ";
```

Comment:
maybe better if u make all members looks the same(all with static keyword or all without)